docs/how-to/coverage-filtering.md

```diff
@@ -0,0 +1,117 @@
+# Coverage Filtering
```

Comment:
This should probably include a comment that this is for an upcoming feature, as this document will be part of the `main` before the uninstrumented target coverage tasks are available.